QuickStartGuide.md

```diff
@@ -0,0 +1,142 @@
+</p>
+
+
+
+## Setup your K8s cluster
+
+<details><summary><sub>Create a K8s cluster, if not available</sub></summary>
+   <sub>
+If you don't have a K8s cluster, you can install a local environment using minikube - (https://minikube.sigs.k8s.io/docs/start/)
+   </sub>
+</details>
+
+:one: Install cert-manager in your cluster. For more information, see [cert-manager](https://cert-manager.io/docs/installation/)
+
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+```
+<sub> *note:- This command uses v1.8.2. Please replace with a later version, if available.</sub>
+
+
+>You may also install cert-manager using Helm. For instructions, see [Installing with Helm](https://cert-manager.io/docs/installation/helm/#installing-with-helm)
+
+
+:two: Next, Generate a Personal Access Token (PAT) for ARC to authenticate with GitHub.
+   - Login to GitHub account and Navigate to https://github.com/settings/tokens/new
+   - Select  **repo**
+   - Click **Generate Token** and then copy the token locally ( we’ll need it later)
+
+
+
+
+## Deploy and Configure ARC
+1️⃣ Deploy  and configure ARC on your K8s cluster. You may use Helm or Kubectl.
+
+
+<details><summary>Helm deployment</summary>
+
+ ##### Create a new file called `custom-values.yaml` containing...
+
+ ```yaml
+authSecret:
+  github_token: REPLACE_YOUR_TOKEN_HERE
+  create: true
+
+```
+<sub> *note:- Replace REPLACE_YOUR_TOKEN_HERE with your PAT that was generated in Step 1 </sub>
+
+##### Add repository
+```shell
+helm repo add actions-runner-controller https://actions-runner-controller.github.io/actions-runner-controller
+```
+
+##### Install Helm chart
+```shell
+helm install -f custom-values.yaml --wait --namespace actions-runner-system \
+  --create-namespace actions-runner-controller \
+  actions-runner-controller/actions-runner-controller
+```
+</details>
+
+<details><summary>Kubectl deployment</summary>
+
+##### Deploy ARC
+```shell
+kubectl apply -f \
+https://github.com/actions-runner-controller/actions-runner-controller/\
+releases/download/v0.22.0/actions-runner-controller.yaml
+```
+<sub> *note:- Replace "v0.22.0" with the version you wish to deploy </sub>
+ 
+
+##### Configure Personal Access Token
+```shell
+kubectl create secret generic controller-manager \
+    -n actions-runner-system \
+    --from-literal=github_token=REPLACE_YOUR_TOKEN_HERE
+````
+<sub> *note:- Replace REPLACE_YOUR_TOKEN_HERE with your PAT that was generated in Step 1 </sub>
+  
+  </details>
+
+2️⃣ Create the GitHub self hosted runners and configure to run against your repository
+
+Create a `runnerdeployment.yaml` file containing
+
+```yaml
+apiVersion: actions.summerwind.dev/v1alpha1
+kind: RunnerDeployment
+metadata:
+  name: example-runnerdeploy
+spec:
+  replicas: 1
+  template:
+    spec:
+      repository: mumoshu/actions-runner-controller-ci
```

Comment:
```suggestion
      repository: mumoshu/actions-runner-controller-ci
```

We might better provide an example repo that contains a simple github actions workflow, so that folks can fork it and use it to test RunnerDeployment. WDYT?

Reply:
Thanks for the comment. Yes, this would help. As discussed, we will add this post merge of this PR incrementally